components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemOffer.vue

```diff
@@ -18,22 +24,39 @@ import { nftToOfferItem } from '@/components/common/shoppingCart/utils'
 import { usePreferencesStore } from '@/stores/preferences'
 import { useMakingOfferStore } from '@/stores/makeOffer'
 import MakeOffer from '@/components/offer/MakeOffer.vue'
+import GalleryItemPriceSection from '@/components/gallery/GalleryItemAction/GalleryItemActionSection.vue'
+import type { NFTOffer } from '@/components/useNft'
 
 const props = defineProps<{
   nft: NFT
+  highestOffer?: NFTOffer
 }>()
-
 const preferencesStore = usePreferencesStore()
 const makeOfferStore = useMakingOfferStore()
+const { doAfterLogin } = useDoAfterlogin()
+const { isCurrentOwner } = useAuth()
+const isOwner = computed(() => isCurrentOwner(props.nft?.currentOwner))
+
+const highestOfferPrice = computed(() => props.highestOffer?.price || '')
 
 const openOfferModal = () => {
   makeOfferStore.clear()
-  const item = nftToOfferItem(props.nft)
+  const item = nftToOfferItem(props.nft, highestOfferPrice.value)
   makeOfferStore.setItem(item)
 
   preferencesStore.setMakeOfferModalOpen(true)
 }
 
+const onMakeOfferClick = () => {
+  doAfterLogin({
+    onLoginSuccess: () => {
+      if (!isOwner.value) {
+        openOfferModal()
+      }
```

Comment:
maybe we can add 

```
else  {
    notify('can't create offer on an item you own')
}
```

Reply:
I would not add the notification message because the nft owner can't see the offer button once he logs in, which is an edge case.

Reply:
got it